src/mlib/str.h

```diff
@@ -587,6 +591,29 @@ mstr_eq (mstr_view left, mstr_view right)
    return memcmp (left.data, right.data, left.len) == 0;
 }
 
+/**
+ * @brief Determine whether two strings are equivalent ignoring case.
+ */
+static inline bool
+mstr_eq_ignore_case (mstr_view left, mstr_view right)
+{
+#ifdef _WIN32
+#define _mstr_strncasecmp _strnicmp
+#else
+#define _mstr_strncasecmp strncasecmp
+#endif
+
+   int got;
+
+   if (left.len != right.len) {
+      return false;
+   }
+   got = _mstr_strncasecmp (left.data, right.data, left.len) == 0;
+
+#undef _mstr_strncasecmp
+   return got;
```

Comment:
You can return the result directly without `got`

Reply:
Done.